index.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Bootstrap CSS -->
+  <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+  <title>Weather Report</title>
+  <link href="styles/index.css" rel="stylesheet" type="text/css"/>
 </head>
 <body>
-    
+    <!-- div main-container will be our container -->
+  <div class="main-container">
+    <header>
+      <h1 class="logo">Weather Report</h1>
+      <h3 class="sub"> For the lovely city of</h3>
+      <h2 id="default-city">City Name</h2>
+    </header>
+    <main>
+      <div id="widget-container">
+        <section class="temperature-section">
+          <h3>Temperature</h3>
+          <button id="up-arrow">⬆️</button>
+          <button id="down-arrow">⬇️ </button>
+          <div id="temperature">70</div>
```

Comment:
Consider initializing this with a class `orange` so that the temperature is orange when the page first loads